fuzzers/libafl-fuzz/src/main.rs

```diff
@@ -0,0 +1,246 @@
+#![feature(io_error_more)]
+#![deny(clippy::pedantic)]
+#![allow(clippy::unsafe_derive_deserialize)]
```

Comment:
Copy&paste the massive amount of flags we use in libafl/lib.rs

Reply:
Why? I would rather add it only if necessary and so far it is not

Reply:
You'll be forced to write cleaner code through annoying lints

Reply:
clean code is good :saluting_face: